### PR TITLE
Added documentation for MelCloud components.

### PR DESCRIPTION
**Pull request by o0Zz**
**Description:**
Add documentation for melcloud component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/17338):** https://github.com/home-assistant/home-assistant/pull/17338

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.melcloud.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## {% linkable_title Configuration %}
+
+To enable the platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: melcloud
+    login: YOUR@EMAIL.COM
+	password: yourpassword
```
Comment (frenck) [inline source/_components/climate.melcloud.markdown]: - There is a <TAB> character in here.
- Please use `YOUR_PASSWORD` as per documentation standards.

File: source/_components/climate.melcloud.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+comments: false
+sharing: true
+footer: true
+logo: mitsubishi.png
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Local Polling"
+---
+
+
```
Comment (frenck) [inline source/_components/climate.melcloud.markdown]: Remove extra empty line.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by o0Zz** [comment]
@Frenck, everthing should be resolved.

File: source/_components/climate.melcloud.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: melcloud
+    login: YOUR@EMAIL.COM
+    password: YOUR_PASSWORD
+```
+
+{% configuration %}
+login:
```
Comment (Danielhiversen) [inline source/_components/climate.melcloud.markdown]: This is called email in the code

File: source/_components/climate.melcloud.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  description: Login used on https://app.melcloud.com/ website
+  required: true
+  type: string
+password:
+  description: Password used on https://app.melcloud.com/ website
+  required: true
+  type: string
+{% endconfiguration %}
+
```
Comment (Danielhiversen) [inline source/_components/climate.melcloud.markdown:52]: `language` and `lease_time` are missing

File: source/_components/climate.melcloud.markdown [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## {% linkable_title Configuration %}
+
+To enable the platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: melcloud
+    login: YOUR@EMAIL.COM
```
Comment (Danielhiversen) [inline source/_components/climate.melcloud.markdown]: This is called email in the code

**Review by Danielhiversen — CHANGES_REQUESTED** [review]

File: source/_components/climate.melcloud.markdown [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+description: "Instructions on how to integrate MelCloud supported Mitsubishi AC with Home Assistant."
+date: 2018-10-14 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mitsubishi.png
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Local Polling"
```
Comment (Danielhiversen) [inline source/_components/climate.melcloud.markdown]: Is this not cloud polling?

File: source/_components/climate.melcloud.markdown (hunk 4 above)
Comment (o0Zz) [reply source/_components/climate.melcloud.markdown:52]: Language as been remove from code

File: source/_components/climate.melcloud.markdown [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+email:
+  description: Login used on https://app.melcloud.com/ website
+  required: true
+  type: string
+password:
+  description: Password used on https://app.melcloud.com/ website
+  required: true
+  type: string
+lease_time:
+  description: How long time the melcloud cache stay valid in seconds (Default: 60)
```
Comment (tjorim) [inline source/_components/climate.melcloud.markdown:48]: Add `default: 60` to `lease_time` [per the docs](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration), the default value should not be put in the description.

File: source/_components/climate.melcloud.markdown [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+To enable the platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: melcloud
+    email: YOUR@EMAIL.COM
+    password: YOUR_PASSWORD
+    lease_time: 60
```
Comment (tjorim) [inline source/_components/climate.melcloud.markdown:35]: Please omit this line. The example entry should only contain required values (and 60 is the default anyway).

**Review by tjorim — CHANGES_REQUESTED** [review]
Thanks for this docs PR! I left some small notes.

File: source/_components/climate.melcloud.markdown (hunk 7 above)
Comment (tjorim) [reply source/_components/climate.melcloud.markdown:48]: Apparently this also made the deploy preview fail because of the space behind the colon.

**Comment by frenck** [comment]
Parent PR has been closed, closing this PR as well.

**Pull request closed without merging**